### ISSUE TITLE
Installation Errors

### ISSUE THREAD

**Issue by GeorgePatsias** [klezVirus/inceptor#2]
git clone --recurse https://github.com/klezVirus/inceptor.git
Produces the following errors
```
Cloning into 'inceptor'...
remote: Enumerating objects: 272, done.
remote: Counting objects: 100% (272/272), done.
remote: Compressing objects: 100% (203/203), done.
remote: Total 272 (delta 56), reused 272 (delta 56), pack-reused 0
Receiving objects: 100% (272/272), 15.58 MiB | 1.19 MiB/s, done.
Resolving deltas: 100% (56/56), done.
Submodule 'inceptor/obfuscators/powershell/chameleon' (git@github.com:klezVirus/chameleon.git) registered for path 'inceptor/obfuscators/powershell/chameleon'
Cloning into 'C:/Users/user/Desktop/inceptor/inceptor/obfuscators/powershell/chameleon'...
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:klezVirus/chameleon.git' into submodule path 'C:/Users/user/Desktop/inceptor/inceptor/obfuscators/powershell/chameleon' failed
Failed to clone 'inceptor/obfuscators/powershell/chameleon'. Retry scheduled
Cloning into 'C:/Users/user/Desktop/inceptor/inceptor/obfuscators/powershell/chameleon'...
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:klezVirus/chameleon.git' into submodule path 'C:/Users/user/Desktop/inceptor/inceptor/obfuscators/powershell/chameleon' failed
Failed to clone 'inceptor/obfuscators/powershell/chameleon' a second time, aborting
```

python update-config.py
Produces the following errors
```
Traceback (most recent call last):
  File "C:\Users\user\Desktop\inceptor\inceptor\update-config.py", line 325, in <module>
    update_config()
  File "C:\Users\user\Desktop\inceptor\inceptor\update-config.py", line 55, in update_config
    c = Config()
  File "C:\Users\user\Desktop\inceptor\inceptor\config\Config.py", line 16, in __init__
    self.rebase()
  File "C:\Users\user\Desktop\inceptor\inceptor\config\Config.py", line 38, in rebase
    for key, directory in self.config["DIRECTORIES"].items():
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.9_3.9.1776.0_x64__qbz5n2kfra8p0\lib\configparser.py", line 963, in __getitem__
    raise KeyError(key)
KeyError: 'DIRECTORIES'
```

**Comment by Mantraufo** [comment]
I present the same errors and I tried to fix it manually so that it does not use the missing folders but it still gives errors

(venv) C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor>python update-config.py
Traceback (most recent call last):
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\update-config.py", line 325, in <module>
    update_config()
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\update-config.py", line 55, in update_config
    c = Config()
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\config\Config.py", line 16, in __init__
    self.rebase()
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\config\Config.py", line 38, in rebase
    for key, directory in self.config["DIRECTORIES"].items():
  File "c:\users\lenovo\appdata\local\programs\python\python39\lib\configparser.py", line 963, in __getitem__
    raise KeyError(key)
KeyError: 'DIRECTORIES'


(venv) C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor>python inceptor.py dotnet jhjkhkjhkjhhjhjhjhkjh.exe -o lol.exe
Traceback (most recent call last):
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\inceptor.py", line 229, in <module>
    generator = DotNetArtifactGenerator(file=binary_abs_path,
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\generators\DotNetArtifactGenerator.py", line 46, in __init__
    chain.push(HexEncoder())
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\encoders\HexEncoder.py", line 14, in __init__
    super().__init__()
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\encoders\Encoder.py", line 26, in __init__
    self.implementations_dir: Path = Config().get_path("DIRECTORIES", "encoders")
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\config\Config.py", line 16, in __init__
    self.rebase()
  File "C:\Users\LENOVO\Documents\NoteMalware\inceptor\inceptor\config\Config.py", line 38, in rebase
    for key, directory in self.config["DIRECTORIES"].items():
  File "c:\users\lenovo\appdata\local\programs\python\python39\lib\configparser.py", line 963, in __getitem__
    raise KeyError(key)
KeyError: 'DIRECTORIES'

**Comment by klezVirus** [comment]
Hi there, Thanks for your submission, I've replicated your issue and fixed with commit [faded2693f0e2ac7d2b34755d9678006db42fa8d](https://github.com/klezVirus/inceptor/commit/faded2693f0e2ac7d2b34755d9678006db42fa8d). I'm closing this now, let me know if there is any other blocking error.
